src/Tickets/Admin/Tickets/Page.php

```diff
@@ -282,27 +462,4 @@ public function get_link_to_edit_posts() {
 		// Create link to edit posts page.
 		return add_query_arg( [ 'post_type' => $post_type ], admin_url( 'edit.php' ) );
 	}
-
-	/**
-	 * Render the `All Tickets` page.
-	 *
-	 * @since 5.14.0
-	 *
-	 * @return void
-	 */
-	public function render_tec_tickets_admin_tickets_page() {
-		tribe_asset_enqueue_group( 'event-tickets-admin-tickets' );
-
-		/** @var Tribe__Tickets__Admin__Views $admin_views */
-		$admin_views = tribe( 'tickets.admin.views' );
-
-		$context = [
-			'tickets_table'  => tribe( List_Table::class ),
-			'page_slug'      => static::$slug,
-			'tickets_exist'  => static::tickets_exist(),
-			'edit_posts_url' => $this->get_link_to_edit_posts(),
-		];
-
-		$admin_views->template( 'admin-tickets', $context );
```

Comment:
Why are we moving this into a template file? Usually we try to keep logic and context out of template files and only pass the context vars.

Reply:
Nope, you are correct. Not sure what I was thinking there. Corrected.